sdks/java/io/azure-cosmosdb/build.gradle

```diff
@@ -0,0 +1,73 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * License); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+  id("org.apache.beam.module")
+  id("scala")
+}
+
+ext {
+  junitVersion = "5.9.1"
+  cosmosVersion = "4.37.1"
+  cosmosContainerVersion = "1.17.5"
+  //lombokVersion = "1.18.24"
+  bsonMongoVersion = "4.7.2"
+  log4jVersion = "2.19.0"
+}
+
+applyJavaNature(automaticModuleName: "org.apache.beam.sdk.io.azure.cosmosdb")
+
+description = "Apache Beam :: SDKs :: Java :: IO :: Azure Cosmos DB"
+ext.summary = "IO library to read and write Azure Cosmos DB services from Beam."
+
+dependencies {
+  implementation("org.scala-lang:scala-library:2.13.10")
```

Comment:
This dependency may be worth putting into `BeamModulePlugin.groovy`. Scala versions have to match across all modules a users includes, right?

Reply:
I'm downgrade to version 2.12.17, and the scala version is varied, example:

```
fd -x rg -nH scala-library
./sdks/java/io/sparkreceiver/2/build.gradle:50:    compileOnly "org.scala-lang:scala-library:2.11.12"
./sdks/java/io/azure-cosmosdb/build.gradle:38:  implementation("org.scala-lang:scala-library:2.12.17")
./runners/spark/spark_runner.gradle:161:    compileOnly "org.scala-lang:scala-library:2.11.12"
./runners/spark/spark_runner.gradle:165:    compileOnly "org.scala-lang:scala-library:2.12.15"
./runners/spark/3/job-server/build.gradle:38:    force "org.scala-lang:scala-library:2.12.10"
./runners/samza/build.gradle:59:  runtimeOnly "org.scala-lang:scala-library:2.11.8"
./sdks/java/io/azure-cosmosdb/build.gradle:38:  implementation("org.scala-lang:scala-library:2.12.17")
./sdks/java/io/sparkreceiver/2/build.gradle:50:    compileOnly "org.scala-lang:scala-library:2.11.12"
./runners/spark/spark_runner.gradle:161:    compileOnly "org.scala-lang:scala-library:2.11.12"
./runners/spark/spark_runner.gradle:165:    compileOnly "org.scala-lang:scala-library:2.12.15"
./runners/spark/3/job-server/build.gradle:38:    force "org.scala-lang:scala-library:2.12.10"
./runners/spark/spark_runner.gradle:161:    compileOnly "org.scala-lang:scala-library:2.11.12"
./runners/spark/spark_runner.gradle:165:    compileOnly "org.scala-lang:scala-library:2.12.15"
./runners/spark/3/job-server/build.gradle:38:    force "org.scala-lang:scala-library:2.12.10"
./runners/spark/3/job-server/build.gradle:38:    force "org.scala-lang:scala-library:2.12.10"
./runners/spark/3/job-server/build.gradle:38:    force "org.scala-lang:scala-library:2.12.10"
./runners/samza/build.gradle:59:  runtimeOnly "org.scala-lang:scala-library:2.11.8"
./runners/samza/build.gradle:59:  runtimeOnly "org.scala-lang:scala-library:2.11.8"
./sdks/java/io/sparkreceiver/2/build.gradle:50:    compileOnly "org.scala-lang:scala-library:2.11.12"
./sdks/java/io/azure-cosmosdb/build.gradle:38:  implementation("org.scala-lang:scala-library:2.12.17")
./sdks/java/io/sparkreceiver/2/build.gradle:50:    compileOnly "org.scala-lang:scala-library:2.11.12"
./sdks/java/io/sparkreceiver/2/build.gradle:50:    compileOnly "org.scala-lang:scala-library:2.11.12"
./sdks/java/io/azure-cosmosdb/build.gradle:38:  implementation("org.scala-lang:scala-library:2.12.17")
./sdks/java/io/azure-cosmosdb/build.gradle:38:  implementation("org.scala-lang:scala-library:2.12.17")
./sdks/java/io/sparkreceiver/2/build.gradle:50:    compileOnly "org.scala-lang:scala-library:2.11.12"
./ownership/JAVA_DEPENDENCY_OWNERS.yaml:967:  org.scala-lang:scala-library:
./ownership/JAVA_DEPENDENCY_OWNERS.yaml:969:    artifact: scala-library
./ownership/JAVA_DEPENDENCY_OWNERS.yaml:967:  org.scala-lang:scala-library:
./ownership/JAVA_DEPENDENCY_OWNERS.yaml:969:    artifact: scala-library
```

what would be nice is cross compilation so that it compiles for all stable versions (2.12, 2.13 3.x), I hope to help with this later.